manifests/config.pp

```diff
@@ -70,5 +99,25 @@
       notify  => Class['confluence::service'],
     }
 
+    file { "${confluence::webappdir}/confluence/WEB-INF/web.xml":
+      source  => 'puppet:///modules/confluence/web.xml',
+      mode    => '0600',
+      owner   => $::confluence::user,
+      group   => $::confluence::group,
+      require => Class['confluence::install'],
+      notify  => Class['confluence::service'],
+    }
+
+  }
+  # if JDBC was configured along with the license key and server_id, skip some server setup steps.
+  if $confluence::tomcat_jdbc_settings and $confluence::license and $confluence::server_id {
```

Comment:
I think this needs to be `!empty($tomcat_jdbc_settings)` if it's a hash? Otherwise, it'll be true even when the hash is empty.